substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -1624,21 +1787,16 @@ pub mod pallet {
         ///
         /// Refunds the transaction fees upon successful execution.
         #[pallet::call_index(20)]
-        #[pallet::weight(T::WeightInfo::reap_stash(*num_slashing_spans))]
+        #[pallet::weight(<T as Config>::WeightInfo::reap_stash(*num_slashing_spans))]
         pub fn reap_stash(
-            origin: OriginFor<T>,
+            _origin: OriginFor<T>,
             stash: T::AccountId,
             num_slashing_spans: u32,
         ) -> DispatchResultWithPostInfo {
-            let _ = ensure_signed(origin)?;
-
-            let ed = T::Currency::minimum_balance();
-            let reapable = T::Currency::total_balance(&stash) < ed
-                || Self::ledger(Self::bonded(stash.clone()).ok_or(Error::<T>::NotStash)?)
-                    .map(|l| l.total)
-                    .unwrap_or_default()
-                    < ed;
-            ensure!(reapable, Error::<T>::FundedTarget);
+            ensure!(
+                T::Currency::total_balance(&stash) == T::Currency::minimum_balance(),
+                Error::<T>::FundedTarget
+            );
```

Comment:
Looks like we missed this change.  Our version is from Substrate v3.0.0 (before the monthly releases).  I would prefer that we use the newer Substrate logic here, but just change the ` < ed` to ` <= ed`.